x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -256,7 +261,7 @@ export function getWaterfall(
         case 'span':
```

Comment:
We no longer need the `filter` above (because we don't get error documents any longer). And the `default` case below is probably not that useful anymore (it seems overly defensive)